CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - yum-epel-4.5.0 (from yum-epel-4.1.2)
 - Disable `aws-ubuntu-eni-helper` service, available in Deep Learning AMIs, to avoid conflicts with `configure_nw_interface.sh` when configuring instances with multiple network cards.
 - Set MTU to 9001 for all the network interfaces when configuring instances with multiple network cards.
+- Remove the trailing dot from the FQDN in the file /etc/hosts of the compute nodes
```

Comment:
This is a change more than an Enhancement.
+ missing period at the end.
+ use `/etc/hosts`

Reply:
Done